include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -298,7 +298,55 @@ class InterfaceKinetics : public Kinetics
     */
     double interfaceCurrent(const size_t iphase);
 
+    /**
+     * Set/modify derivative settings. For the interface kinetics object this
+     * includes `"skip-coverage-dependence"`, `"skip-electrochem"`, and `"rtol-delta"`.
+     *
+     *  - `skip-coverage-dependence` (boolean) ... if `true` (default), rate constant
+     *    coverage dependence is not considered when evaluating derivatives.
+     *  - `skip-electrochem` (boolean) ... if `true` (default), electrical charge is
+     *    not considered in evaluating the derivatives and these reactions are treated
+     *    as normal surface reactions.
+     *   - `rtol-delta` (double) ... relative tolerance used to perturb properties
+     *     when calculating numerical derivatives. The default value is 1e-8.
+     *
+     * @param settings  AnyMap containing settings determining derivative evaluation.
+     */
```

Comment:
`InterfaceKinetics::getDerivativeSettings` is missing at the moment (it needs to be overloaded). Note that the boolean flags are set reverse to what is used in `GasKinetics` (all defaults are `false` there, i.e. accurate Jacobians are returned by default). 

I'd also suggest to move the docstring to `Kinetics.h` where flags relevant to `GasKinetics` are documented.

Reply:
@ischoegl I think default false makes more sense, I had it set to true as the functionality of these specific types of functions is not presently supported.